e2e/tests/nightly/annual-access-renewal.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import AccessRenewalPage from 'app/page/access-renewal-page';
+import ProfilePage from 'app/page/profile-page';
+import ProfileConfirmationModal from 'app/modal/profile-confirmation-modal';
+import Navigation, { NavLink } from 'app/component/navigation';
+import HomePage from 'app/page/home-page';
+
+// Important: the access test user must be in a state where they are currently failing access renewal
+// due to an expired "profile last confirmed" date.  This is accomplished by the CircleCI nightly test runner
+// executing the API project.rb CLI tool SetAccessModuleTimestamps
+
+describe('Annual Access Renewal', () => {
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.ACCESS_TEST_ACCESS_TOKEN_FILE);
+  });
+
+  test('Expired User is redirected to Annual Access Renewal (AAR) on login', async () => {
```

Comment:
Note: there is an implicit order dependency between these two test cases, since the latter is destructive. FWIW in-order execution seems guaranteed by jest: https://jestjs.io/docs/setup-teardown#order-of-execution-of-describe-and-test-blocks